base/src/main/java/com/smartdevicelink/proxy/rpc/OnVehicleData.java

```diff
@@ -335,6 +335,11 @@
 public class OnVehicleData extends RPCNotification {

```

Comment:
@kostyaBoss in the HTML section above you are missing <tr>
 *      <td>climateData</td>
 *      <td>ClimateData</td>
 *      <td>See ClimateData</td>
 *      <td>N</td>
 *      <td></td>
 *      <td>
 *         @since SmartDeviceLink 7.1.0
 *      </td>
 *  </tr>

Please add it.